jwst/extract_1d/extract_1d_step.py

```diff
@@ -172,7 +171,7 @@ class Extract1dStep(Step):
     center_xy = float_list(min=2, max=2, default=None)  # IFU extraction x/y center
     ifu_autocen = boolean(default=False) # Auto source centering for IFU point source data.
     bkg_sigma_clip = float(default=3.0)  # background sigma clipping threshold for IFU
-    ifu_rfcorr = boolean(default=False) # Apply 1d residual fringe correction
+    ifu_rfcorr = boolean(default=True) # Apply 1d residual fringe correction
```

Comment:
One concern with changing this to True is that it would then require failsafe cases for all non-MRS uses of extract1d, which may have unintended consequences.  May be easier to keep as False, since the only purpose of the flag now is to print a message saying how the flag isn't necessary any longer.

Reply:
I have not added any print statement that says the flag is not necessary. Do we want that still ?. Since we are keeping the flag it seemed like something that might confuse the user. It never hurts to set it to True and we are not going to remove the ifu_rfcorr now. But I can add a flag that says 'By default ifu_rfcorr is true now on need to set it.' If you think it will help


Since the rf_corr is only done in ifu.py (ifu_extract1d) it only operates on MIRI and NIRSPEC. I turned off the ifu_rfcorr for all data but MIR_MRS in extract_1d_step (so NIRSPEC data).
That should do it. I ran it on a few NIRSpec IFU files I had and it works as expected.
 But I will run the regression tests to make sure

Reply:
Given the approach we're now taking I think this change is ok without print statements, but let's add 'for MIRI MRS data' to the comment to make clear it only affects that mode.